intel-deflater.html

```diff
@@ -1,34 +0,0 @@
----
----
-<!doctype html>
-<html>
-	{% include head.html title="Intel Deflater" %}
-	<body>
-		<div class="wrapper">
-			{% include header.html %}
-
-			<h2>Intel Deflater</h2>
```

Comment:
We should keep this but update it to reflect how IntelDeflater is used from the GKL package -- and possibly generalize it to mention other compression/decompression capabilities available from GKL are used in Picard. 

@lbergelson, do you know who would be in the best position to update/write about this?